jiant/scripts/download_data/datasets/nlp_tasks.py

```diff
@@ -0,0 +1,129 @@
+"""Use this for tasks that can be obtained from NLP without further/special processing"""
+
+import jiant.scripts.download_data.utils as download_utils
+import jiant.utils.python.io as py_io
+
+
+NLP_CONVERSION_DICT = {
+    # === GLUE === #
+    "cola": {
+        "path": "glue",
+        "name": "cola",
+        "field_map": {"sentence": "text"},
```

Comment:
did `nlp` change the name of the field from the original dataset?

Reply:
(For some of these, including CoLA) the datasets do not have column headers. So the naming is arbitrary on both our/HF's parts.)

For now, I'm mapping to the field names that the Task objects are expecting. We can consider switching over to using HF's naming conventions in a later PR, but this PR should not touch any of the task implementations.